test/helpers/Fixtures.sol

```diff
@@ -384,4 +389,67 @@ contract DeploymentFixtures is StdCheats, DeploymentHelpers {
             wq.onOracleReport(false, 0, 0);
         }
     }
+
+    function hugeDeposit() internal {
+        // It's impossible to process deposits if withdrawal requests amount is more than the buffered ether,
+        // so we need to make sure that the buffered ether is enough by submitting this tremendous amount.
+        handleStakingLimit();
+        handleBunkerMode();
+
+        address whale = address(1);
```

Comment:
address(1) is a precompile, I'd suggest using something like 100500 instead to avoid some unexpected behavior.